plugins/web/ui/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # The install dir of the UI root relative to the install prefix of VAST itself.
-set(ui_install_dir "${CMAKE_INSTALL_DATADIR}/vast/plugin/web/www")
+set(ui_install_dir "${CMAKE_INSTALL_DATADIR}/vast/plugin/web/www/"
+)# TODO REVIEW do we need the trailing slash?
```

Comment:
If anything we should need it only in the `file(COPY ... DESTINATION ...)`, but not in this variable, which we also use in other places that definitely do not like the trailing slash.

Reply:
Seems like this is what was creating the nested www directories. 🤷

Reply:
I implemented a more appropriate fix to this by appending the trailing slashes where required.